src/js/async-loader.js

```diff
@@ -11,7 +11,7 @@ const asyncLoader = (imports, callback = ([cmp]) => cmp) => {
 export default {
     loadVulnerabilities: asyncLoader([
         import('@red-hat-insights/insights-frontend-components/components/Vulnerabilities'),
-        import('./vulnerabilitiesStyles')
+        import('./vulnerabilityStyles')
```

Comment:
Do we want to change this?

Reply:
I renamed the file as well, so nothing should be breaking. Is renaming it an issue?